vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -17903,6 +17980,7 @@ quantitykind:Turns
 .
 quantitykind:UniversalGasConstant
```

Comment:
Suggest we move this to the PhysicalConstant graph

Reply:
In performing this task I found out that we already have it: constant:MolarGasConstant. I made the change anyway, but we should decide which if not both we want to keep.

Reply:
Molar has qkdv:A-1E0L2I0M1H-1T-2D0 and value of 8.314472 (should be 8.31446261815324 according to Wikipedia). Universal should have the same DV but doesn't. I suggest we keep them both, fix the Universal DV reference, and have it point to the existing constant:Value_MolarGasConstant. Also add a qudt:exactMatch relation between the two.

...which makes me notice that we still have some skos:exactMatch relations sprinkled in our vocabs that we should change to qudt:exactMatch.

Reply:
I have made these changes. I ran a query looking for skos:exactMatch on all things below qudt:Concept and found several but they are all quantity kinds. Do you want them fixed?

Reply:
I changed the value to the one on wikipedia.

Reply:
I am changing the skos:exactMatch values to qudt:exactMatch. I have found one, for ElementaryCharge, that points to quantitykind:AtomicUnitOfCharge, that appears to not exist. When finished I will look into that further.

Reply:
There is no quantitykind:AtomicUnitOfCharge in the vocabulary.

Reply:
...but there is a constant:AtomicUnitOfCharge, which makes sense.

There is also constant:ElementaryCharge
I suggest those both should live only in the constants file, related by qudt:exactMatch.
Currently they each have separate constant:Value_... instances, so probably not worth having them both point to the same one.

In addition, I think it is valid to keep the unit:ElementaryCharge instance, because it is used as a unit of measure. (Recall my point that any constant can be used as a unit of measure, but only some are actually used that way).

We should remove references to quantitykind:AtomicUnitOfCharge

Reply:
I will try to change from qk to c for AtomicUnitOfCharge, though when I load the qk graph the constants are not referenceable so I'll have to do it by editing the text version.

Reply:
I haven't done anything but change from qk:AUOC to constant:AUOC on qudt:exactMatch and removed the skos:exactMatch because the other comments you made are outside the scope of this PR (as a matter of fact we have ventured pretty far off that PR). My query is no longer finding any skos:exactMatch occurrences.